saleor/graphql/webhook/mutations.py

```diff
@@ -200,3 +204,29 @@ def perform_mutation(cls, _root, info, **data):
                 )
 
         return super().perform_mutation(_root, info, **data)
+
+
+class WebhookDeliveryRetry(BaseMutation):
```

Comment:
After consideration, I think we should rename it to `EventDeliveryRetry`, which would be more consistent with the naming of the types.